lib/ecto/adapters/postgres/connection.ex

```diff
@@ -1074,6 +1087,10 @@ if Code.ensure_loaded?(Postgrex) do
     defp tagged_to_db(:id), do: "bigint"
     defp tagged_to_db(:integer), do: "bigint"
     defp tagged_to_db({:array, type}), do: [tagged_to_db(type), ?[, ?]]
+
+    defp tagged_to_db({type, size}) when is_number(size),
+      do: [ecto_to_db(type), ?(, to_string(size), ?)]
+
```

Comment:
In the meantime, I tried to implement a type casting with the possibility of adding a size parameter. I will revert my last commit if we won't allow this.

Reply:
I think if we do this we can only accept `{:bitstring}` size and not `:bitstring` anywhere in Ecto. Including schema field types. This is because `type(^value, s.bitstring_field)` will never work. But that's a big API decision so want to see Jose's thoughts.

Reply:
Also we wouldn't accept it for arbitrary types. So I think let's revert this and move forward after Jose has had a chance to catch up :)

Reply:
Sure no problem, everything reverted. Sorry for this.

Reply:
@Gigitsu Doing some more digging, I found we can cast to type `varbit` without specifying the number of bits. So I think we can leave things as is and just change ecto_to_db to use `varbit`